DiMuons/src/EleHelper.cc

```diff
@@ -108,8 +147,10 @@ pat::ElectronCollection SelectEles( const edm::Handle<edm::View<pat::Electron>>&
   }
 
   if ( _ele_ID.find("veto")   == std::string::npos && _ele_ID.find("loose") == std::string::npos && 
-       _ele_ID.find("medium") == std::string::npos && _ele_ID.find("tight") == std::string::npos )
-    std::cout << "Ele ID is neither tight, medium, loose, nor tight: " << _ele_ID
+       _ele_ID.find("medium") == std::string::npos && _ele_ID.find("tight") == std::string::npos &&
+       _ele_ID.find("wp90") == std::string::npos && _ele_ID.find("wpLoose") == std::string::npos &&
+       _ele_ID.find("tZq") == std::string::npos)
```

Comment:
```suggestion
       _ele_ID.find("tZq") == std::string::npos && _ele_ID.find("none") == std::string::npos )
```

Reply:
This check should also include "none" as possible choice otherwise it sends too many unwanted warning.
Do I understand correctly?

Reply:
Fixed this problem by changing the warning message to include 'none' as an option. Also reworded the warning message. Commit is here https://github.com/UFLX2MuMu/Ntupliser/pull/101/commits/6f655b83f04a37679bf014201a69b19b63fdc3c2.